src/components/popup/collectible-assets.tsx

```diff
@@ -23,7 +24,7 @@ export const CollectibleAssets = memo((props: StackProps) => {
         const { assetName, contractName } = getAssetStringParts(key);
         return (
           <AssetItem
-            amount={collectible.count}
+            amount={new BigNumber(collectible.count)}
```

Comment:
this should really be changed at the source, https://github.com/blockstack/stacks-wallet-web/blob/d53dcac4d62d0c58307ae35e1d10855541289c22/src/components/popup/collectible-assets.tsx#L11, https://github.com/blockstack/stacks-wallet-web/blob/d53dcac4d62d0c58307ae35e1d10855541289c22/src/common/api/accounts.ts#L14

Reply:
Reverted back to passing asset item a string so not needed here anymore